tools/wheels/install-librdkafka.sh

```diff
@@ -19,7 +20,19 @@ echo "$0: Installing librdkafka $VER to $DEST"
 [[ -d "$DEST" ]] || mkdir -p "$DEST"
 pushd "$DEST"
 
-curl -L -o lrk$VER.zip https://www.nuget.org/api/v2/package/librdkafka.redist/$VER
+# Check if variable exists
+if [ -z "${GITHUB_TOKEN}" ]; then
+    echo "Error: GITHUB_TOKEN is not set"
+    exit 1
+fi
+
+curl -H "Authorization: Bearer ${GITHUB_TOKEN}" \
```

Comment:
Downloads appropriate patched version of librdkafka from our NuGet packages index.